code/controllers/subsystem/processing/processing.dm

```diff
@@ -1,40 +1,65 @@
 //Used to process objects. Fires once every second.
 
-var/datum/subsystem/processing/SSprocessing
 /datum/subsystem/processing
 	name = "Processing"
-	priority = 25
-	flags = SS_BACKGROUND|SS_POST_FIRE_TIMING|SS_NO_INIT
-	wait = 10
+	can_fire = FALSE
 
 	var/stat_tag = "P" //Used for logging
-	var/list/processing = list()
-	var/list/currentrun = list()
+	var/list/processing_list = list()	//what's processing
+	var/list/run_cache = list()	//what's left to process in the next run
+	var/delegate	//what the processing call is
 
 /datum/subsystem/processing/New()
-	NEW_SS_GLOBAL(SSprocessing)
+	if(type == /datum/subsystem/processing)
+		flags |= SS_NO_FIRE	//this SS should be derived, but MC will create it anyway
```

Comment:
thought, add a SS_ABSTRACT flag, and have the mc check it and delete any subsystem that uses it when it inits them.